hedera-node/src/test/java/com/hedera/test/mocks/TestUsagePricesProvider.java

```diff
@@ -66,30 +68,40 @@ public void loadPriceSchedules() {
 	}
 
 	@Override
-	public FeeData activePrices() {
+	public Map<SubType, FeeData> activePrices() {
 		throw new UnsupportedOperationException();
 	}
 
 	@Override
-	public FeeData pricesGiven(HederaFunctionality function, Timestamp at) {
+	public FeeData defaultActivePrices() {
```

Comment:
Shouldn't this be implemented>?

Reply:
I haven't implemented this yet, because the `ActivePrices()` method above also throws an `UnsupportedOperationException` (Was like this before this PR)